Objects/call.c

```diff
@@ -942,22 +948,14 @@ PyObject_CallFunction(PyObject *callable, const char *format, ...)
 PyObject *
 PyEval_CallFunction(PyObject *callable, const char *format, ...)
```

Comment:
commit message: this change also optimizes PyEval_CallFunction() and PyEval_CallMethod() to avoid temporary tuple when possible.